includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1460,20 +1439,112 @@ protected function build_shipping_methods( $shipping_methods ) {
 		return $shipping;
 	}
 
+
+	/**
+	 * Builds the response object to pass to Payment Request
+	 *
+	 * @since 5.3.0
+	 *
+	 * @param bool   $shipping_pending True is the user hasn't defined yet a shipping address.
+	 * @param string $page             Page where the request was made. It can be `product_page` or `cart_page`.
+	 *
+	 * @return array Data object used to update Payment Request button on the client.
+	 */
+	protected function build_response( $shipping_pending = true, $page = 'product_page' ) {
+		if ( 'product_page' == $page ) {
+			$itemized_display_items = true;
+		} elseif ( 'cart_page' == $page ) {
+			$itemized_display_items = false;
+		}
```

Comment:
I'm not sure if replacing `$itemized_display_items` from the function parameters and replacing it with `$page` makes sense, to be honest. It's only used to then tell if we should show itemized displayItems, right?

There are a couple reasons why I think it was better the other way:
1. Other than product and cart, there is also the checkout page where the PRB can be displayed, which might be a little confusing if it's missing from here.
2. `$itemized_display_items` has a meaning in this context. It might be a little overkill to pass `$page` to this function only then to map to an `$itemized_display_items` boolean and not use `$page` anywhere else. It's also easier to reason about passing a boolean to `build_response` if we check the function signature expecting a variable named `$itemized_display_items` as a parameter.
3. It's less code and less noise.

Reply:
Yeah. I think you are right. `$itemized_display_items` seemed like a proper variable name. I'll revert the change.